test/tests/functional/pbs_que_resc_usage.py

```diff
@@ -167,3 +173,36 @@ def test_resources_assigned_with_zero_val(self):
         q_status = self.server.status(QUEUE, id='workq')
         self.assertNotIn('resources_assigned.foo',
                          q_status[0].keys(), self.err_msg)
+
+    @skipOnCpuSet
+    def test_resources_assigned_deletion(self):
+        """
+        Test resources_assigned.<resc_name> deletion from the system
+        """
+        # create a resource
+        self.create_custom_resc()
+        # submit jobs
+        j_attr = {ATTR_queue: 'workq',
+                  'Resource_List.select': '1', 'Resource_List.foo': '3'}
+        j1 = Job(TEST_USER, j_attr)
+        j1.set_sleep_time(30)
+        jid_1 = self.server.submit(j1)
+        j2 = Job(TEST_USER, j_attr)
+        j2.set_sleep_time(30)
```

Comment:
You aren't waiting for the job to finish, no need to set the sleep time.

Reply:
Actually, here we are waiting for jobs to end after "except".